pkg/apis/migration/v1alpha1/miganalytic_types.go

```diff
@@ -0,0 +1,110 @@
+/*
+Copyright 2019 Red Hat Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	"k8s.io/apimachinery/pkg/api/resource"
+
+	kapi "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// MigAnalyticSpec defines the desired state of MigAnalytic
+type MigAnalyticSpec struct {
```

Comment:
@jmontleon after thinking on this for a while longer, due to the way that k8s allows us to look up resources, it will respond to

```
oc get miganalytics
```

Which makes sense and is inline with desired naming. I have no idea what would happen with miganalysis on the other hand. Does k8s know how to handle plurals that end in 's'? A mystery for the ages. 

My **updated** vote is to keep as-is